NGit.Test/NGit.Test.csproj

```diff
@@ -1,310 +1,21 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>9.0.21022</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{E1FA8F29-5BAB-4FB7-8362-B8CE67ECF7C2}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <RootNamespace>NGit.Test</RootNamespace>
-    <AssemblyName>NGit.Test</AssemblyName>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
-    <TargetFrameworkProfile />
+    <TargetFramework>net461</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <OutputPath>..\bin</OutputPath>
```

Comment:
Why are we using a non-standard output path? It's not good to build all tests into the same top-level output folder. Also, at some point we'll want to run the tests on frameworks other than net461.